components/footer/footer.html

```diff
@@ -0,0 +1,81 @@
+<footer class="footer-container">
+  <div
+    class="inner-container container d-flex flex-column"
+    style="max-width: 1440px"
+  >
+    <div class="row footer-container--contact">
+      <div class="col-lg-4 col-12 footer-container--contact--subscribe">
+        <div data-include="./components/logoBox/logoBox.html"></div>
+        <div
+          class="footer-container--contact--subscribe--container d-flex flex-column"
+        >
+          <p class="footer-container--contact--subscribe--container--info">
+            Subscribe to email alerts. We promise not to spam your inbox.
+          </p>
```

Comment:
the space between this p and input equal 14 px

Reply:
no, it is equal 14px.